config/initializers/002-rails_failover.rb

```diff
@@ -44,7 +44,7 @@
 
       # Test connection to the master, and trigger master failover if needed
       ActiveRecord::Base.connected_to(role: ActiveRecord.writing_role) do
-        ActiveRecord::Base.connection.active?
+        ActiveRecord::Base.connection.connect!.active?
```

Comment:
Does `#active?` do anything for us here anymore, or is the new `#connect!` enough?

Reply:
Yes, we still need `#active?`. `#connect!` has been introduced in Rails 7.1 and its implementation is roughly the same as doing `.tap(&:verify!)`. I’m not even sure we need this change, but we did it in multisite and failover, I think, so better safe than sorry :sweat_smile:  Actually, I should probably make this change in a separate PR as it’s not related to Rails 7.2 directly.